apps/emqx_bridge_s3tables/src/emqx_bridge_s3tables_aggreg_partitioned.erl

```diff
@@ -75,19 +76,22 @@ fill(Records, #iceberg{} = Container0) ->
     ),
     fill_partitions(Partitions, Container0).
 
--spec close(container()) -> {partition_out(), write_metadata()}.
-close(#iceberg{partitions = Partitions}) ->
+-spec close(container()) -> partition_out().
+close(#iceberg{partitions = Partitions} = Container) ->
```

Comment:
Nit
`iceberg` record name here seems too specific; the rest of the module is not Iceberg-specific.